docusaurus/docs/dev-docs/migration/v4/migration-guide-4.6.1-to-4.7.0.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Migrate from 4.6.1+ to 4.7.0
+description: Learn how you can migrate your Strapi application from 4.6.1+ to 4.7.0.
+displayed_sidebar: devDocsSidebar
+---
+
+import BuildCommand from '/docs/snippets/build-npm-yarn.md'
+import DevelopCommand from '/docs/snippets/develop-npm-yarn.md'
+
+# v4.6.1+ to v4.7.0 migration guide
+
+The Strapi v4.6.1+ to v4.7.0 migration guide upgrades v4.6.1+ to v4.7.0. We [describe in 1-2 sentences what we introduced in 4.7.0 and why the migration is needed] 
+<!-- TODO: fill sentence above -->
+
+The migration guide consists of:
+
+<!-- TODO: describe "additional steps" below -->
+- Upgrading the application dependencies
+- additional step 1 specific to this migration_
+- additional step 2 specific to this migration_
+- …
+- Reinitializing the application
+
+## Upgrading the application dependencies to 4.7.0
+
+:::prerequisites
+Stop the server before starting the upgrade.
+:::
+
+1. Upgrade all of the Strapi packages in `package.json` to `4.7.0`:
+
+   ```json title="package.json"
+
+   {
+     // ...
+     "dependencies": {
+       "@strapi/strapi": "4.7.0",
+       "@strapi/plugin-users-permissions": "4.7.0",
+       "@strapi/plugin-i18n": "4.7.0"
+       // ...
+     }
+   }
+   ```
+
+2. Save the edited `package.json` file.
+
+3. Run either `yarn` or `npm install` to install the new version.
+
+:::tip
+If the operation doesn't work, try removing your `yarn.lock` or `package-lock.json`. If that doesn't help, remove the `node_modules` folder as well and try again.
+:::
+
+<!-- TODO: update title -->
+## Heading title for specific step 1
+
+<!-- TODO: fill in this section -->
+Quickly describe the nature of the change.
+Give a procedure to make the change:
+
+1. do this
+2. do this
+3. …
+
+<!-- TODO: add code example if relevant and update title property -->
+```jsx title="/path/to/file.js"
+// my beautiful code here
+```
```

Comment:
```suggestion
```jsx title="/config/admin.js"
module.exports = ({ env }) => ({
  // ...
  transfer: {
    token: {
      salt: env('TRANSFER_TOKEN_SALT'),
    },
  },
});

```
```

Reply:
Oh, forgot to add the ts version